bitnami/nats/templates/metrics-svc.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.metrics.enabled }}
```

Comment:
What would be the difference between `metrics-svc.yaml` and `monitoring-svc.yaml`?

```yaml
## NATS svc used for monitoring
## ref: https://github.com/nats-io/gnatsd#monitoring
##
monitoring:
  service:
    ## Kubernetes service type
    type: ClusterIP
    port: 8222
    ## Specify the nodePort value for the LoadBalancer and NodePort service types.
    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
    ##
    # nodePort:
    ## Provide any additional annotations which may be required. This can be used to
    ## set the LoadBalancer service type to internal only.
    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
    ##
    annotations: {}
    ## Use loadBalancerIP to request a specific static IP,
    ## otherwise leave blank
    ##
    loadBalancerIP:
```

Reply:
The [NATS monitoring service](https://docs.nats.io/nats-server/configuration/monitoring) provides statistics

the [NATS metric exporter](https://github.com/bitnami/bitnami-docker-nats-exporter) (already included) takes those statistics and provides Prometheus metrics

Reply:
https://github.com/nats-io/prometheus-nats-exporter

Reply:
Ok, I see!